### ISSUE TITLE
Consider expanding into saving datasets

### ISSUE THREAD

**Issue by bquistorff** [wbuchanan/readit#5]
I love the idea of this package, since aside from the benefit of aggregating across multiple files, it in general allows reading data from so many formats using existing Python tools. Do you know of anything similar for saving files into various formats using Python in Stata? If not, would you like to collaborate on that with me? My concrete interest is to improve the ability to use parquet files in Stata. There is [stata-parquet](https://github.com/mcaceresb/stata-parquet), but it's only for Linux and the installation/usage is rather advanced. I think using Python would be easier to develop, install, and use (though at the cost some performance for large files).

**Comment by wbuchanan** [comment]
Hi @bquistorff,

Long time no see.  In terms of saving to different file formats, I would probably rely primarily on what already exists in Pandas for that.  The write capabilities are definitely a bit more limited, but it doesn't always seem like there is the same demand for writing to multiple file formats.  I think to create the test sets I used StatTransfer to create the different file formats, but it's been a while since I worked on this.  That said, I could be up for collaborating on something.
